tests/unit/test_req.py

```diff
@@ -602,10 +602,10 @@ def test_parse_editable_local_extras(
 def test_exclusive_environment_markers():
     """Make sure RequirementSet accepts several excluding env markers"""
     eq26 = install_req_from_line(
```

Comment:
I would also change the name of the variable to `eq36`.

Reply:
Nice catch! ^>^